jetty-security/src/main/java/org/eclipse/jetty/security/SecurityHandler.java

```diff
@@ -76,11 +77,8 @@
 
     static
     {
-        for (Authenticator.Factory factory : ServiceLoader.load(Authenticator.Factory.class))
-        {
-            __knownAuthenticatorFactories.add(factory);
-        }
-
+        TypeUtil.serviceStream(ServiceLoader.load(Authenticator.Factory.class))
+            .forEach(__knownAuthenticatorFactories::add);
```

Comment:
Ignoring the failure would let jetty-all with jetty-jaspi work (see #4621)

Perhaps this should be backported to jetty 9.4.x?